ambari-infra/pom.xml

```diff
@@ -25,7 +25,7 @@
 
   <properties>
     <jdk.version>1.8</jdk.version>
-    <solr.version>7.5.0</solr.version>
+    <solr.version>7.7.0</solr.version>
```

Comment:
can you search for other occurrences 7.5.0 solr version? (also in migration scripts / containers etc.) also you can update this for ambari-logsearch

Reply:
done